src/coreComponents/constitutive/fluid/PVTFunctions/PureWaterProperties.cpp

```diff
@@ -0,0 +1,276 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file PureWaterProperties.cpp
+ */
+
+#include "constitutive/fluid/PVTFunctions/PureWaterProperties.hpp"
+
+namespace geosx
+{
+
+using namespace stringutilities;
+
+namespace constitutive
+{
+
+namespace PVTProps
+{
+
+TableFunction const *
+PureWaterProperties::makeViscosityTable( string const & functionName,
```

Comment:
Just for the sake of consistency - maybe name this function also mentioning that it's saturated water?
```suggestion
PureWaterProperties::makeSaturationViscosityTable( string const & functionName,
```

Reply:
Thanks @mkhait, I modified the code as you suggested.